jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
```

Comment:
probably want this off by default

Reply:
Indeed, nice catch, leftover from debugging stuff.